### PR TITLE
Task/eparker71/tlt1870/implement breadcrumbs

### PR DESCRIPTION
**Pull request by eparker71**
added breadcrumbs to the dashboard, Import Isites Content, manage people, and manage sections tools. In manage section I moved the manage-people.css file from the icommons_ui project into the manage sections tool. I needed to make a change to a style and it seemed more appropriate to have the tool specific css located in the tool project. I wanted the breadcrumbs to have the same look and feel in each tool and the dashboard, as well as be in the same page location so it looks to the end user like they are all one app. 

I did not remove the manage_people.css file from the icommons_ui project, I added it as a tech debt item (TLT-2627)

### CODE REVIEW

File: canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html [hunk 1]
```diff
@@ -16,17 +16,19 @@
   {% endif %}
 {% endblock stylesheets %}
 
-{% block content %}
 
+{% block breadcrumb %}
+<nav>
+    <h3>
```
Comment (elliottyates) [inline canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html]: Can we make this respect the header type (h1) and classes that UX provided? (see removed lines 25-27 below)

File: manage_people/templates/manage_people/results_list.html [hunk 2]
```diff
@@ -3,7 +3,15 @@
 {% load manage_people_tags %}
 {% load static %}
 
-{% block breadcrumbs %}<a href="{% url 'manage_people:user_form' %}">Manage People</a> <i class="fa fa-chevron-right"></i> Add People{% endblock breadcrumbs %}
+{% block breadcrumbs %}
+<nav>
+    <h3>
+        <a href="{% url 'dashboard_course' %}">Manage Course</a>&nbsp;<i class="fa fa-chevron-right"></i>&nbsp;
+        <a href="{% url 'manage_people:user_form' %}">Manage People</a>&nbsp;<i class="fa fa-chevron-right"></i>&nbsp;
+        <a href="{% url 'manage_people:find_user' %}">Search for people</a>&nbsp;<i class="fa fa-chevron-right"></i>&nbsp;Add People
```
Comment (elliottyates) [inline manage_people/templates/manage_people/results_list.html]: Why did this hierarchy change? It used to just be `user_form`, looks like we inserted `find_user` in there too?

File: manage_people/static/manage_people/css/manage_people.css [hunk 3]
```diff
@@ -0,0 +1,1080 @@
+/*
```
Comment (elliottyates) [inline manage_people/static/manage_people/css/manage_people.css:1]: what a monster! :)

File: manage_people/templates/manage_people/base_new.html [hunk 4]
```diff
@@ -7,8 +7,8 @@
     <link href="{% static "favicon.ico" %}" rel="icon" type="image/x-icon">
     {% block stylesheet %}
         {% block fonts %}{% endblock fonts %}
-        <link rel="stylesheet" type="text/css" href="{% static "lti-css/manage-people.css" %}" media="screen"/>
         <link rel="stylesheet" type="text/css" href="{% static "font-awesome-4.5.0/css/font-awesome.min.css" %}" media="screen"/>
+        <link rel="stylesheet" type="text/css" href="{% static "manage_people/css/manage_people.css" %}" media="screen"/>
```
Comment (elliottyates) [inline manage_people/templates/manage_people/base_new.html:11]: Hmm, why not merge base.css and manage_people.css?

File: canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html (hunk 1 above)
Comment (elliottyates) [inline canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html]: (that would also mean there's no need for the new h3 styling added to manage_people.css)

Comment (elliottyates) [inline canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html]: (this change will be needed on all pages, looks like)

File: manage_people/static/manage_people/css/manage_people.css [hunk 5]
```diff
@@ -0,0 +1,1080 @@
...
+}
+.list-with-floats .float-right {
+  padding-top: 0.25em;
+}
+ul#existing_enrollments_list {
+  margin-bottom: 1em;
+}
+a#find_another_user {
+  margin-right: 1em;
+}
```
Comment (sapnamysore) [inline manage_people/static/manage_people/css/manage_people.css:1080]: Do we need everything in this file for manage people styles? i.e. did we want only MP tool specific stuff in this css?

File: manage_sections/templates/manage_sections/section_details.html [hunk 6]
```diff
@@ -6,17 +6,21 @@
 <link rel="stylesheet" type="text/css" href="{% static "manage_sections/css/manage_sections.css" %}" media="screen"/>
 {% endblock stylesheet %}
 
+{% block breadcrumb %}
+<nav>
+    <h3>
+        <a href="{% url 'dashboard_course' %}">Manage Course</a>&nbsp;<i class="fa fa-chevron-right"></i>
+        <a href="{% url 'manage_sections:create_section_form' %}">&nbsp;Manage Sections</a>&nbsp;<i class="fa fa-chevron-right"></i>
+        <span>{{section_name}} (<span class="enroll_count"></span>)</span>
```
Comment (elliottyates) [inline manage_sections/templates/manage_sections/section_details.html:14]: For really long ones, we may want to ensure that the line spacing/padding is what UX is happy with.

**Comment by elliottyates** [comment]
A couple small questions, but looks good!

File: canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html (hunk 1 above)
Comment (elliottyates) [inline canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html]: (removing all these h1 / styling comments about breadcrumbs after our offline discussion)

File: manage_people/static/manage_people/css/manage_people.css (hunk 5 above)
Comment (eparker71) [inline manage_people/static/manage_people/css/manage_people.css:1080]: This file was called manage_people.css in the icommons_ui project. I just moved a copy to the local project. I don't know If there are styles in there that are no longer needed. That's outside the scope of this story.

File: manage_people/templates/manage_people/base_new.html (hunk 4 above)
Comment (eparker71) [inline manage_people/templates/manage_people/base_new.html:11]: yeah, that makes sense.

File: manage_sections/templates/manage_sections/section_details.html (hunk 6 above)
Comment (eparker71) [inline manage_sections/templates/manage_sections/section_details.html:14]: sure, we can iterate if it becomes an issue.

File: manage_people/templates/manage_people/results_list.html (hunk 2 above)
Comment (eparker71) [inline manage_people/templates/manage_people/results_list.html]: `Manage Course >  Manage People >  Search for people > Add People` The text at the end "Add People" may need to change, but this is the breadcrumb trail when you go through the process of adding a person. We can change it if needed once we get some feedback.

Comment (elliottyates) [inline manage_people/templates/manage_people/results_list.html]: The 'Search for people' link is not in the original tool -- did the ACs change? I must be missing something.

Comment (elliottyates) [inline manage_people/templates/manage_people/results_list.html]: (I kinda like the extra level of breadcrumb though -- my vote is to keep it, but maybe it makes the header / breadcrumb line too long?...)

Comment (eparker71) [inline manage_people/templates/manage_people/results_list.html]: I see what you mean. There is a result page that we could have the same breadcrumb as the search page. There is a link on that page people can use to return to the search page. Let me show what I have to vittorio and see what he thinks.

**Pull request merged**